watertap/unit_models/electrodialysis_1D.py

```diff
@@ -439,11 +521,17 @@ def build(self):
             units=pyunits.meter,
             doc="The width of the electrodialysis cell, parallel to the current/ion flux direction",
         )
-        self.spacer_thickness = Var(
+        self.channel_height = Var(
```

Comment:
Should the corresponding change be made in 0D?

Reply:
Same here about mirroring the changes to 0d.